spec/fixtures/multiline-soa

```diff
@@ -0,0 +1,7 @@
+osuosl.org      IN SOA  ns1.auth.osuosl.org. hostmaster.osuosl.org. (
```

Comment:
@knightsamar can you provide an example of a test case you'd like to be included?